tests/integration_tests/clouds.py

```diff
@@ -66,8 +74,9 @@ def launch(self, user_data=None, launch_kwargs=None,
                 "\n".join("{}={}".format(*item) for item in kwargs.items())
             )
         )
-        pycloudlib_instance = self.cloud_instance.launch(**kwargs)
-        pycloudlib_instance.wait(raise_on_cloudinit_failure=False)
+
+        pycloudlib_instance = self._perform_launch(kwargs)
```

Comment:
I don't love what I did here. I basically carved out this base method to add a place I could call "lxd init" instead of "lxd launch" in the overridden method. It's the easiest way to get it done, but the `launch()` and then `_perform_launch()` feels a little forced. I'm open to different ideas.